widget/playground/src/globalStyles.ts

```diff
@@ -1,7 +1,33 @@
 import { globalCss } from '@rango-dev/ui';
 
 export const globalStyles = globalCss({
-  body: {
-    fontFamily: '$primary',
-  },
+  body: { fontFamily: '$primary' },
 });
+
```

Comment:
I don't get the points of this comments.

Reply:
removed.